examples/toolchains/cc/WORKSPACE

```diff
@@ -0,0 +1,38 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "rules_cc",
+    sha256 = "4dccbfd22c0def164c8f47458bd50e0c7148f3d92002cdb459c2a96a68498241",
+    urls = [
+        "https://github.com/bazelbuild/rules_cc/releases/download/0.0.1/rules_cc-0.0.1.tar.gz",
+    ]
+)
+
+local_repository(
```

Comment:
Please add a comment indicating that this should be replaced by an `http_archive` as per the [setup instructions](https://github.com/tweag/rules_nixpkgs/#setup) for a regular use-case outside the rules_nixpkgs repository. That should make it easier for people relatively new to Bazel to understand how to use this.

Same for the other `WORKSPACE` files.